js/lib/lib.go

```diff
@@ -18,15 +18,19 @@
  *
  */
 
+//go:generate rice embed-go
+
 package lib
 
 import (
 	"github.com/GeertJohan/go.rice"
 	"github.com/dop251/goja"
 )
 
-var CoreJS *goja.Program = goja.MustCompile(
-	"core-js/shim.js",
-	rice.MustFindBox("core-js").MustString("client/shim.js"),
-	true,
-)
+func GetCoreJS() *goja.Program {
+	return goja.MustCompile(
+		"core-js/shim.js",
```

Comment:
Not sure why, but `js/lib/rice-box.go` contains what seems like all of the files in the `core-js` folder when it should be (only?) the minified shim code. Running `grep -E "^\s+Filename:" js/lib/rice-box.go | wc -l` gives me `2036` files and the `rice-box.go` file itself is over 5 MB...

Reply:
looking at the `shim.js` it is just a bunch of `require` I guess `go.rice` understand it and import every file that is required.

Reply:
Hmm I'm not sure if go.rice works like that. In any case, embedding only the minified version would get us a smaller git repo and binary and probably faster parsing by goja.